lib/Installation/Partitioner/ExpertPartitionerPage.pm

```diff
@@ -109,4 +115,30 @@ sub press_accept_button {
     send_key('alt-a');
 }
 
+sub open_clone_partition_dialog {
+    my ($self) = @_;
+    assert_screen(EXPERT_PARTITIONER_PAGE);
+    send_key($self->{partition_table_shortcut});
+    for (1 .. 2) { wait_screen_change { send_key "down" } }
```

Comment:
can you replace the "magic numbers" here with some variables? What if we want to clone different number of disks?

Reply:
This for {1..2} is not for disks, as indicates the function's name open_clone_partition_dialog: It is for the menu "partition table" visible at the bottom of the page: http://waaa-amazing.suse.cz/tests/10982#step/raid_gpt/63. I could not use shortcut here as wait_screen_change was seeing some screen change where there were none, even after tweaking the match level.

Reply:
I added a save_screenshot to make it more clear http://waaa-amazing.suse.cz/tests/11043#step/raid_gpt/64

Reply:
As we don't run raid tests in ncurses, we could have used `assert_and_click` call instead.